_posts/2017-04-09-blog_1.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "Blog Week 1"
+date:   2017-04-09
+author: Marianna Brady
+---
+
+I picked a piece of data journalism done by FiveThirtyEight called simply “[Gun Deaths in America] https://fivethirtyeight.com/features/gun-deaths/
```

Comment:
This should be marked up as `[Gun Deaths in America](https://fivethirtyeight.com/features/gun-deaths/)`.